src/Form/input-classifiers.test.js

```diff
@@ -79,17 +116,17 @@ describe('Input Classifiers', () => {
 
     describe('Unified Expiration Date', () => {
         describe.each([
-            { text: 'mm-yyyy', expectedResult: '12-2025' },
-            { text: 'mm/yyyy', expectedResult: '12/2025' },
-            { text: '__-____', expectedResult: '12-2025' },
-            { text: 'mm-yy', expectedResult: '12-25' },
-            { text: 'i.e. 10-2022', expectedResult: '12-2025' },
-            { text: 'MM-AAAA', expectedResult: '12-2025' },
-            { text: 'mm_jj', expectedResult: '12_25' },
-            { text: 'mm.yy', expectedResult: '12.25' },
-            { text: 'mm - yy', expectedResult: '12-25' },
-            { text: 'mm yy', expectedResult: '12 25' },
-            { text: 'ie: 08.22', expectedResult: '12.25' }
+            { text: 'mm-yyyy', expectedResult: '08-2025' },
```

Comment:
I switched to a single-digit month so we can test that the month is properly padded.